### ISSUE TITLE
Twittering-favorite function unbound

### ISSUE THREAD

**Issue by smythp** [hayamiz/twittering-mode#101]
No keybinding for twittering-favorite function.

**Comment by shackra** [comment]
Yup, I noticed this. But that be changed in your Emacs configuration though...

**Comment by smythp** [comment]
Yeah, changed it in mine. But think it's important enough that it deserves
a default binding...it's probably the function I use second or third most
often. I favor C-c f.

Thanks for responding!

**Comment by shackra** [comment]
That's actually the keybinding I use! https://github.com/shackra/.emacs.d/commit/c1174e97bcbe7977118c8b159367e6040fd52fe7
smythp writes:

> Yeah, changed it in mine. But think it's important enough that it deserves
> a default binding...it's probably the function I use second or third most
> often. I favor C-c f.
>
> Thanks for responding!
>
> On Sun, May 3, 2015 at 8:00 PM, Jorge Araya Navarro <
> notifications@github.com> wrote:
>
>> Yup, I noticed this. But that be changed in your Emacs configuration
>> though...
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hayamiz/twittering-mode/issues/101#issuecomment-98558141>
>> .
>>
>
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/hayamiz/twittering-mode/issues/101#issuecomment-98558325

-- 
Pax et bonum.
Jorge Araya Navarro.
ES: Diseñador Publicitario, Programador Python y colaborador en Parabola GNU/Linux-libre
EN: Ads Designer, Python programmer and contributor Parabola GNU/Linux-libre
EO: Anonco grafikisto, Pitino programalingvo programisto kai kontribuanto en Parabola GNU/Linux-libre
https://es.gravatar.com/shackra

**Comment by smythp** [comment]
Great minds think alike, ha. There's a logic to Emacs bindings.
